### PR TITLE
Devstack

### PR DESCRIPTION
**Pull request by klml**
**Description**

our devstack

from our slidedeck https://youtu.be/LXTOdmseUe8?si=tCQk3sWX2CjZd8-Q&t=969

### CODE REVIEW

File: software/jdk.md [hunk 1]
```diff
@@ -3,9 +3,12 @@ Source: SNow
 title: JDK
 developerlink: java.com
 licensingmodel: open source
-logo: /logo/253px-Java-Logo.svg.png
+logo: /logo/320px-OpenJDK_logo.svg.png
 tags:
 - development
+- devstack
+- devstack
```
Comment (FabianWilms) [inline software/jdk.md:10]:
```suggestion
- devstack
```

**Review by FabianWilms — APPROVED** [review]
Minor thing: I don't really like giving these tools a `sortingPriority` of 1, as they will appear first in the list of the "Overview", too. Next to our "flagship" projects, which I think would not be very good.

Having them lexicographically sorted won't suffice?

**Comment by klml** [comment]
@FabianWilms 

> Minor thing: I don't really like giving these tools a sortingPriority of 1, as they will appear first in the list of the "Overview", too. Next to our "flagship" projects, which I think would not be very good.

u r right! 

> Having them lexicographically sorted won't suffice?

whats about offset 100?

**Comment by FabianWilms** [comment]
> whats about offset 100?

defintely better but

> Having them lexicographically sorted won't suffice?

Or worded differently: why do you want to prioritize some parts?

**Comment by klml** [comment]
@FabianWilms 

>  Or worded differently: why do you want to prioritize some parts?

imho Vue.js is more important than openshift

tl;dr to have the same order like on our standard devstack slide https://www.youtube.com/watch?si=tCQk3sWX2CjZd8-Q&t=969&v=LXTOdmseUe8&feature=youtu.be
